apis/go.mod

```diff
@@ -113,5 +113,5 @@ require (
 // must consistent within modules and service operators
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20240830023148-b7d0481c9094 //allow-merging
 
-// custom RabbitmqClusterSpecCore for OpenStackControlplane (v2.6.0_patches_tag)
+// custom RabbitmqClusterSpecCore for OpenStackControlplane (v2.9.0_patches_tag)
 replace github.com/rabbitmq/cluster-operator/v2 => github.com/openstack-k8s-operators/rabbitmq-cluster-operator/v2 v2.6.1-0.20241017142550-a3524acedd49 //allow-merging
```

Comment:
its might be kind of confusing that this shows v2.6.1-... but `a3524ac` is the latest commit from the `v2.9.0_patches` branch

Reply:
agree. I just did the 'go get' from the head of the new branch and this is what go mod included